src/goose/console/server.clj

```diff
@@ -0,0 +1,42 @@
+(ns goose.console.server
+  (:require
+    [goose.brokers.redis.broker :as redis]
+    [goose.console.handler :as handler]
+    [ring.adapter.jetty :as jetty]
+    [ring.middleware.file :refer [wrap-file]]
+    [ring.middleware.resource :refer [wrap-resource]]
+    [ring.util.response :as response]
+    [goose.defaults :as d]))
+
+(defonce server (atom nil))
+
+(defn routes [{:keys [uri broker]}]
+  (if (= uri "/")
+    (handler/landing-page {:broker broker})
+    (response/response "<h1>Not Found</h1>")))
+
+(defn wrap-broker [handler broker]
+  (fn [req]
+    (handler (assoc req :broker broker))))
+
+(defn middleware [broker]
+  (-> routes
+      (wrap-broker broker)
+      (wrap-resource "public")
+      (wrap-file "resources/public")))
+
+(defn start-server [{:keys [broker
```

Comment:
question: Is "broker" a suitable name to be used in console? Or a better name like "producer" or "data-store" is better suited? 
@olttwa